docs/Getting-Started/Create-bootable-media/bootable-usb-using-linux.md

```diff
@@ -0,0 +1,111 @@
+
+Use this guide to create Pop!_OS bootable USB media from Linux:
+
+- Install ISO flashing software.
+- Download the Pop!_OS.iso file.
+- Verify the ISO.
+- Flash the ISO to your USB storage device.
+
+---
+## I. Install Popsicle
+
+![Popsicle Icon](/images/create-bootable-usb-linux/popsicle-icon.png)
+
+Create a bootable USB drive for Pop!_OS using Popsicle. Popsicle is a Linux utility for flashing multiple USB devices in parallel, written in Rust. You can install Popsicle via flatpak or AppImage.
+### AppImage
+
+1. Download the Popsicle AppImage [from this repo.](https://github.com/pop-os/popsicle/releases/latest)
+
+2. Navigate to the downloaded AppImage. Right-click the appimage and select Properties.
+
+3. Select the `Permissions` tab and check the `Execute` box.
+
+![AppImage Execute Permission](/images/create-bootable-usb-linux/appimage-execute.png)
+
+4. Double click the Popsicle AppImage to launch it.
+### Flatpak
+
+Linux users with [Flatpak installed](https://flatpak.org/setup/) in their distribution can simply download and install the [Popsicle flatpak](https://flathub.org/apps/details/com.system76.Popsicle) from Flathub. 
+
+
+## II. Download the Pop!_OS.iso File
+
+![Linux Download ISO](/images/create-bootable-usb-linux/using-linux-download-iso.png)
+
+Download the appropriate Pop!_OS.iso file for your target system. See the [preparation guide](create-bootable-usb.md#standard-nvidia--ras-pi-4-isos) for information about each version.
+
+## III. Verify the Pop!_OS.iso File
+
+You should verify that your download matches the checksum before trying to install. This ensures that you've received the full, complete download and that it is not corrupted. Compare the hash values generated using the below commands to the values shown on the [Pop!_OS download page](https://pop.system76.com/). **Do not use the image below to verify hash values.**
+
+![Compare Hash Values](/images/create-bootable-usb-linux/compare-hash-values.png)
+
+When entering the appropriate checksum command below, ensure:
+
+- The file name in your command matches the downloaded ISO in your local directory. 
+- The examples below use the ~/Downloads folder. Make sure the appropriate explicit or relative path is used in your command.
+
+>**Note**: Popsicle also provides a feature for easily checking the SHA256 checksum of an ISO file (Step 3). Skip this section if you prefer a GUI method.
+### Verify the Intel/AMD ISO
+
+```bash
+sha256sum Downloads/pop-os_21.10_amd64_intel_7.iso
+```
+
+### Verify the NVIDIA ISO
+
+```bash
+sha256sum Downloads/pop-os_21.10_amd64_nvidia_7.iso
+```
+
+### Verify the RASP PI 4 ISO
+
+```bash
+sha256sum Downloads/pop-os_21.10_arm64_raspi_7.img.xz
+```
+
+## IV. Create the Bootable USB
+
+Flash your USB media with the Pop!_OS.iso image. Note that this process will completely erase all previous data on the USB storage device.
+
+1. Launch Popsicle. The app may display as USB Flasher in search results.
```

Comment:
I didn't realize until this point that the numbers in ordered lists aren't showing up on the rendered page for me. We'll need to fix that styling.

Reply:
I'm previewing with the `mdbook serve` command. The numbers are showing up for me. Are you previewing using npm? 
![image](https://user-images.githubusercontent.com/80931038/161158991-8f6ecd36-f18e-4a63-935b-134c9b531feb.png)

Reply:
The issue might only be with the light theme:

![image](https://user-images.githubusercontent.com/7199422/161333522-d8815733-c3c2-43cd-ae1a-84aa76f10653.png)

Just now, it wasn't happening until I switched to the dark theme and then back to the light theme. I'll try to figure out what's going on with it. Maybe something to fix in a separate PR (along with improving the dark theme.)

Reply:
It looks like our main.js file is empty. According to the mdbook docs this could be why our default theme is navy: https://rust-lang.github.io/mdBook/format/theme/index.html

Compare to the [Redox mdbook](https://doc.redox-os.org/book/ch02-04-preparing-the-build.html) which appears to have all the default themes: 
![image](https://user-images.githubusercontent.com/80931038/161338813-e682d820-6cbb-4cd3-a0ea-618eed2c8076.png)

Reply:
I don't think `main.js` specifically has to do with it (I don't see that file mentioned in that mdbook docs page, and we don't have that in the tech-docs repo), but I see where the themes are. I'm going to work on it a bit.

Reply:
This is being addressed in https://github.com/pop-os/docs/pull/5.